### PR TITLE
Sprinting on some changes!

### PR DESCRIPTION
**Pull request by AlannaBurke**
I'm bringing this back to life! Started with some minor text edits, and am working on adding and editing some of the content. Wanted to make sure the review was manageable, so I'll submit it periodically as I go. Documentation is my jam now, so I want this to be a fantastic guide, and I'm also going to focus on making it as readable as possible.

### CODE REVIEW

File: manuscript/introduction/contributors.md [hunk 1]
```diff
@@ -9,6 +9,25 @@ When adding documents, be sure to use [GitHub Flavored Markdown](https://guides.
...
 
+- alannaburke
+- drnikki
+- yesct
+- cleverington
+- bradleyfields
+
+#### Editors
+
+- cottser## Initial Notes to Contribute
```
Comment (laughnan) [inline manuscript/introduction/contributors.md]: do we need a new line here for the `Initial Notes...` heading?

File: manuscript/common-exclusion-pitfalls/venue.md [hunk 2]
```diff
@@ -1,38 +1,62 @@
...
+Things to consider when choosing venues for your camp, conference, after party, or even local attractions to suggest:
 
 ## Accessibility
-When choosing Venues, especially for Event locations, try and ensure buildings meet ADA Compliance Standards.
+When choosing venues, especially for event locations, try to ensure buildings meet ADA compliance standards.
 
-* Mobility
-  * Have few stairs, and have alternatives to support all potential Attendees. Doors or entrances that are easily opened (maybe wide, maybe automatic). Consider those that can't walk (wheelchairs), those with difficulty walking, those who have pain while walking or standing. Not all disabilities are visible.
+### Mobility
+* Have few stairs, and have alternatives to support all potential attendees. Doors or entrances that are easily opened (thik wide, automatic). Consider those who can't walk (wheelchairs), those with difficulty walking, those who have pain while walking or standing. Not all disabilities are visible.
```
Comment (laughnan) [inline manuscript/common-exclusion-pitfalls/venue.md]: `thik` should be `think`?

**Review by laughnan — COMMENTED** [review]
this looks amazing!! 😍  thanks for these changes!

File: manuscript/introduction/contributors.md (hunk 1 above)
Comment (AlannaBurke) [reply manuscript/introduction/contributors.md]: This section is all weird - i have it fixed in some newer changes but i can't figure out how to bring them into my current PR. Forked PRs seem to be weird.

File: manuscript/common-exclusion-pitfalls/venue.md (hunk 2 above)
Comment (AlannaBurke) [reply manuscript/common-exclusion-pitfalls/venue.md]: This is fixed in my next round of changes!

**Review by laughnan — APPROVED** [review]

**Pull request merged**